man/systemd.timer.xml

```diff
@@ -51,7 +51,9 @@
     but simply left running. There is no concept of spawning new service instances in this case. Due to this, services
     with <varname>RemainAfterExit=</varname> set (which stay around continuously even after the service's main process
```

Comment:
It'd be nicer to say `with <varname>RemainAfterExit=true</varname>` here.

Reply:
Added the `true`